internal/service/project/resource_project.go

```diff
@@ -301,12 +320,13 @@ func (r *projectRS) Create(ctx context.Context, req resource.CreateRequest, resp
 	if resp.Diagnostics.HasError() {
 		return
 	}
-
+	tags := *NewAdminTags(ctx, projectPlan.Tags)
```

Comment:
it's normally risk to take the content without checking nil reference, i guess in this case we know NewAdminTags always returns a non-nil pointer, but it somebody changes NewAdminTags later that could not be true anymore

Reply:
fixed in 35d729e4e186219cab55b274cb1af334b82e1645